platform/broadcom/sonic-platform-modules-delta/ag5648/scripts/led_control

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+FAN1_EEPROM="-y 3 0x51 0x0a"
+FAN2_EEPROM="-y 3 0x52 0x0a"
+FAN3_EEPROM="-y 3 0x53 0x0a"
+FAN4_EEPROM="-y 3 0x54 0x0a"
```

Comment:
I do not know what this means. Instead of using i2cget to get these values, these values should be exposed by platform drivers to operate.

Reply:
I use the fan trays eeprom to check fan trays insert or not.